### PR TITLE
Handle deals that were removed from observed firestore query

### PR DESCRIPTION
**Pull request by paweljanicki**
## What was done
Handle deals removed from a query and delete them from deals map if the user is not representative/PL

#### After

### CODE REVIEW

File: src/services/DataSourceDealsTypes.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { IDealDiscussion } from "entities/DealDiscussions";
 import { IDealRegistrationTokenSwap } from "entities/DealRegistrationTokenSwap";
 import { IDealTokenSwapDocument } from "./../entities/IDealTypes";
 import { Address } from "./EthereumService";
+import { IDocumentUpdates } from "./FirestoreTypes";
```
Comment (hiaux0) [inline src/services/DataSourceDealsTypes.ts:7]: `DataSourceDealsTypes.ts` is supposed to be a generic interface.
Is there another way to specify this type? (and not add a dependency on firestore)

Comment (dkent600) [reply src/services/DataSourceDealsTypes.ts:7]: agreed

Comment (dkent600) [reply src/services/DataSourceDealsTypes.ts:7]: IDocumentUpdates  should be defined here, not in FirestoreTypes

**Review by dkent600 — CHANGES_REQUESTED** [review]

**Comment by paweljanicki** [comment]
Decision was made to refactor the code in a way that those changes are not relevant anymore

**Pull request closed without merging**